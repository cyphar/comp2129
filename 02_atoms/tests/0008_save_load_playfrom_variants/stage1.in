LOAD save0.as
PLAYFROM 1000000
STAT
DISPLAY
UNDO
DISPLAY
UNDO
DISPLAY
