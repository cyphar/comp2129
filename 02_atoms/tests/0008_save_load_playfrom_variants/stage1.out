Game Loaded

Game Ready
Red's Turn

Player Red:
Grid Count: 1

Player Green:
Grid Count: 1

Player Purple:
Grid Count: 1

Player Blue:
Grid Count: 1

Player Yellow:
Grid Count: 1


+-----------------------+
|  |  |G1|  |  |  |  |  |
|R1|  |  |P1|  |  |  |  |
|  |  |B1|  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |Y1|
+-----------------------+

Yellow's Turn


+-----------------------+
|  |  |G1|  |  |  |  |  |
|R1|  |  |P1|  |  |  |  |
|  |  |B1|  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
+-----------------------+

Blue's Turn


+-----------------------+
|  |  |G1|  |  |  |  |  |
|R1|  |  |P1|  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
+-----------------------+

